jgiven-gradle-plugin/src/test/groovy/com/tngtech.jgiven.gradle/JGivenPluginShould.groovy

```diff
@@ -54,14 +56,15 @@ class JGivenPluginShould extends Specification {
         new File(testProjectDir.root, 'build').deleteDir()
         result = GradleRunner.create()
                 .withProjectDir(testProjectDir.getRoot())
-                .withArguments("--build-cache", "test", "jgivenTestReport", "-S")
+                .withArguments("--build-cache", "test", "jgivenTestReport", "-S", "--info")
                 .withPluginClasspath()
                 .build()
 
 
         then:
         result.task(":test").outcome == FROM_CACHE
         result.task(":jgivenTestReport").outcome == FROM_CACHE
+        //TODO: caching is disabled, because there is an unmonitored implicit dependency on the "Test" task, probably the dependency getResults() needs to be changed to a "Provider<File>"  The actual insertion of the file into the task needs to be altered as well.
```

Comment:
I don't understand what the TODO means about caching is disabled

Reply:
Well gradle detects an implicit dependency on the test task that it did not consider when ordering the tasks. As such gradle ignores the cached output and executes the task every time.